controls/scheduler/appointments/overview.md

```diff
@@ -38,4 +38,5 @@ The image below shows the appointments in WeekView:
 
 - [Appointment Template]({%slug scheduler-appointment-template%})
 - [Appointments Styling]({%slug scheduler-appointments-styling%})
-- [Recurrence]({%slug scheduler-recurrence-overview %}) 
\ No newline at end of file
+- [Recurrence]({%slug scheduler-recurrence-overview %})
+- [Non-working hours]({%slug scheduler-net-maui-non-working-hours%})
```

Comment:
* Apply consistent capitalization in all list items (Title Case in this list).
* This suggestion applies to the rest of the articles too.
```suggestion
- [Non-Working Hours]({%slug scheduler-net-maui-non-working-hours%})
```